microsoft/testsuites/kata_conformance/kata_conformance_tests_tool.py

```diff
@@ -0,0 +1,270 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import json
+import os
+from dataclasses import dataclass
+from pathlib import Path, PurePath
+from typing import Any, List, Type
+
+from kubernetes import client, config, utils
+
+from lisa import Environment, notifier
+from lisa.executable import Tool
+from lisa.messages import SubTestMessage, TestStatus, create_test_result_message
+from lisa.testsuite import TestResult
+from lisa.tools import Echo, Git, Go, Make
+from lisa.util import LisaException
+
+
+@dataclass
+class KataConformanceTestResult:
+    name: str = ""
+    status: TestStatus = TestStatus.QUEUED
+
+
+class KataConformanceTests(Tool):
```

Comment:
Follow the name of test suite, like KataTest or KataContainersTest.

Reply:
sure, i will use KataTest